docs/changelog.rst

```diff
@@ -15,6 +15,67 @@ IN DEVELOPMENT
 
 Changes will be summarized here periodically.
 
+New Features:
+
+* Support for Python 3.7 was removed. (Official end-of-life: 2023-06-27).
+  `#5191 <https://github.com/pybind/pybind11/pull/5191>`_
+
+Support for CMake older than 3.15 and some older compilers will also be removed.
+
+Version 2.13.2 (August 13, 2024)
+--------------------------------
+
+New Features:
+
+* A ``pybind11::detail::type_caster_std_function_specializations`` feature was added, to support specializations for
+  ``std::function``'s with return types that require custom to-Python conversion behavior (to primary use case is to catch and
+  convert exceptions).
+  `#4597 <https://github.com/pybind/pybind11/pull/4597>`_
+
+
+.. feat(types)
+
+* Add a special type annotation for C++ empty tuple.
+  `#5214 <https://github.com/pybind/pybind11/pull/5214>`_
+
+
+Bug fixes:
+
+* Make ``gil_safe_call_once_and_store`` thread-safe in free-threaded CPython.
```

Comment:
Could we move these to a new `Modernizations:` section? — Calling these bugs isn't really fair (to ourselves).

Reply:
I don't want to add a new section, and this one specifically is a bug fix. The one below (PyMutex) isn't really a bug fix, and is somewhat important, so I could put it under New Features.

Reply:
We've had a "Changes" section in the past, so I moved several things to that section.